deploy/README.md

```diff
@@ -0,0 +1,10 @@
+# injiweb ui
+
+## Install 
+```
+./install.sh
+```
+
+# Installing DataShare Service : 
+
```

Comment:
Is datashare pr-requisite or dependent service?

In any of the above case where are the instructions to point them to the datashare installation steps.